tests/unit/CoreImageTest.php

```diff
@@ -36,103 +38,336 @@ public function setUp(): void {
 	public function tearDown(): void {
 		// your tear down methods here
 		wp_delete_post( $this->post_id, true );
-		\WPGraphQL::clear_schema();
 
 		parent::tearDown();
+
+		\WPGraphQL::clear_schema();
```

Comment:
We do our actions _before_ we `parent::tearTown()
```suggestion
		\WPGraphQL::clear_schema();

		parent::tearDown();
```

Reply:
Done 🐳